<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInstallXSwiftBusComponent</class>
 <widget class="QFrame" name="CInstallXSwiftBusComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Install xswiftbus</string>
  </property>
  <layout class="QVBoxLayout" name="vl_XSwiftBusComponent">
   <item>
    <widget class="QFrame" name="fr_Info">
     <layout class="QHBoxLayout" name="hl_XSwiftBusIndo" stretch="1,2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="lbl_Explaination">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;xswiftbus&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; is an X-Plane plugin that allows X-Plane and &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;swift&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; to communicate with each other. If you want to use &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;swift&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; with X-Plane you will need to install &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;xswiftbus&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;. Download &lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-style:italic;&quot;&gt;xswiftbus&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; and extract it into the &lt;/span&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline;&quot;&gt;X-Plane/Resources/plugins&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt; folder.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_DownloadAndInstall">
     <property name="title">
      <string>Download and install xswiftbus</string>
     </property>
     <layout class="QGridLayout" name="gl_DownloadAndInstall">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_XSwiftBusPluginDir">
        <property name="text">
         <string>Install directory:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cb_DownloadFile"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="le_XSwiftBusPluginDir">
        <property name="placeholderText">
         <string>X-Plane/Resources/plugins</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_DownloadDir"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Version">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pb_OpenInstallDir">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_DownloadDirectory">
        <property name="text">
         <string>Download directory:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pb_OpenDownloadDir">
        <property name="text">
         <string>open</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="pb_Download">
        <property name="toolTip">
         <string>download and install</string>
        </property>
        <property name="text">
         <string>download</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="tb_DialogInstallDir">
        <property name="toolTip">
         <string>change directory</string>
        </property>
        <property name="statusTip">
         <string>change directory</string>
        </property>
        <property name="icon">
         <iconset resource="../../misc/misc.qrc">
          <normaloff>:/diagona/icons/diagona/icons/folder--pencil.png</normaloff>:/diagona/icons/diagona/icons/folder--pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QToolButton" name="tb_DialogDownloadDir">
        <property name="toolTip">
         <string>change directory</string>
        </property>
        <property name="statusTip">
         <string>change directory</string>
        </property>
        <property name="icon">
         <iconset resource="../../misc/misc.qrc">
          <normaloff>:/diagona/icons/diagona/icons/folder--pencil.png</normaloff>:/diagona/icons/diagona/icons/folder--pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="lbl_Warning">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Warning: &lt;/span&gt;Possible conflicts with plugins using the same libraries &lt;span style=&quot; font-style:italic;&quot;&gt;(e.g. XSquawkBox&lt;/span&gt;). You have to disable those plugins before you can use &lt;span style=&quot; font-style:italic;&quot;&gt;xswiftbus&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_XSwiftBusSettings">
     <property name="title">
      <string>xswiftbus settings</string>
     </property>
     <layout class="QVBoxLayout" name="vl_XSwiftBusSettings">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="BlackGui::Components::CSettingsXSwiftBusComponent" name="comp_SettingsXSwiftBus">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CSettingsXSwiftBusComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/settingsxswiftbuscomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_XSwiftBusPluginDir</tabstop>
  <tabstop>tb_DialogInstallDir</tabstop>
  <tabstop>pb_OpenInstallDir</tabstop>
  <tabstop>le_DownloadDir</tabstop>
  <tabstop>tb_DialogDownloadDir</tabstop>
  <tabstop>pb_OpenDownloadDir</tabstop>
  <tabstop>cb_DownloadFile</tabstop>
  <tabstop>pb_Download</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
